modules/porous_flow/test/tests/actions/tests

```diff
@@ -197,4 +197,13 @@
     issues = '#20260'
     design = 'actions/PorousFlowAddMaterialJoiner.md'
   []
+  [block_restricted]
+    type = 'CSVDiff'
+    input = block_restricted.i
+    threading = '!pthreads'
+    csvdiff = 'block_restricted_out.e'
+    requirement = "PorousFlow shall be able to be block-restricted."
+    issues = '#28030'
+    design = 'actions/PorousFlowFullySaturated.md'
```

Comment:
is there a PorousXYZBase page you could use?

Reply:
I do not fully understand your comment. In the test PorousFlowFullySaturated is used, while the code changes also apply to PorousFlowBasicTHM, and PorousFlowUnsaturated (and PorousFlowActionBase, PorousFlowSinglePhaseBase). Do I have to add all the corresponding *.md files?

Reply:
After re-reading your comment: I think there is no PorousXYZBase.md file present at the moment. I found the following files in [modules/porous_flow/doc/content/source/actions](https://github.com/idaholab/moose/tree/6349167f4073a10e927fc73d403dd65fd370db6c/modules/porous_flow/doc/content/source/actions):

- PorousFlowAddBCAction.md
- PorousFlowAddMaterialAction.md
- PorousFlowAddMaterialJoiner.md
- PorousFlowBasicTHM.md
- PorousFlowFullySaturated.md
- PorousFlowUnsaturated.md

So should I change the line to:
`design = 'actions/PorousFlowFullySaturated.md actions/PorousFlowBasicTHM.md actions/PorousFlowUnsaturated.md'`

Reply:
works for me